internal/core/commands/create_ds.go

```diff
@@ -23,10 +24,11 @@ func (*CreateDeviceStyleCommand) Key() string { return "CreateDeviceStyleCommand
 
 type CreateDeviceStyleCommandHandler struct {
 	Repository repositories.DeviceStyleRepository
+	DDCache    services.DeviceDefinitionCacheService
```

Comment:
if `DDCache` and `Repository` don't need to be public, just keep them starting with lowercase